client/signup/steps/domains/index.jsx

```diff
@@ -380,6 +380,11 @@ class DomainsStep extends Component {
 			Object.assign( { domainItem }, useThemeHeadstartItem )
 		);
 
+		// Skip the plans step if we are in the `domain-transfer` flow.
```

Comment:
Not sure how accurate this description is. Both "transfer" and "connect" look like are part of the same `domain-transfer` flow, one shows the plans while the other doesn't.

Reply:
I think I kinda get it now. This callback could fire from other flows, right? All good in that case.

Reply:
I skipped mentioning the "connect" flow in this comment since we are inside the `handleAddTransfer` method, which handles the transfer part only.